src/Hl7.Fhir.Validation/Schema/ElementSchema.cs

```diff
@@ -15,11 +15,11 @@
 
 namespace Hl7.Fhir.Validation.Schema
 {
-    public class ElementSchema : IAssertion, IMergeable, IGroupValidatable
+    public class ElementSchema : IElementSchema, IMergeable, IGroupValidatable
```

Comment:
When `IElementSchema` is derived from `IGroupValidatable` then `IGroupValidatable`  can be removed here